### PR TITLE
 Documentation change log update

### PR DESCRIPTION
**Pull request by ThomasOliverKimble**
**Description**
Remove R2022b-rev1, reorder and minor corrections in the change-log. See changes [here](https://cyberbotics.com/doc/reference/changelog-r2022?version=documentation-change-log).

Other: I added my latest modifications/big fixes but I believe some are still missing so we should add them either in this PR or in separate ones. We will also have to add the crazyflie when that is merged.

### CODE REVIEW

File: docs/reference/changelog-r2022.md [hunk 1]
```diff
@@ -2,67 +2,44 @@
...
+    - Changed the behavior of `wb_robot_wwi_receive_text` to iterate through the received messages buffer ([#4336](https://github.com/cyberbotics/webots/pull/4336)).
+    - Added a python controller for the [Mavic 2 Pro](../guide/mavic-2-pro.md) demonstrating how to move the drone to specific coordinates and altitude ([#4293](https://github.com/cyberbotics/webots/pull/4293)).
+    - Improved the structure of the [Nao](../guide/nao.md) PROTO: the `version` field changed and the `color` field was replaced with a `customColor` field ([#4180](https://github.com/cyberbotics/webots/pull/4180)).
+    - Allowed negative `scale` values in the [Transform](transform.md) node and added a `ccw` field in the [Mesh](mesh.md) node ([#4243](https://github.com/cyberbotics/webots/pull/4243)).
+    - Added rendering of anchors in joints ([#4256](https://github.com/cyberbotics/webots/pull/4256)).
+    - Replaced ColladaShapes PROTO by the [CadShape](cadshape.md) node ([#4285](https://github.com/cyberbotics/webots/pull/4285)).
+    - Reduced the loading time when texture qualitiy is set to to medium or low ([#4621](https://github.com/cyberbotics/webots/pull/4621)).
```
Comment (ygoumaz) [inline docs/reference/changelog-r2022.md:39]:
```suggestion
    - Reduced the loading time when texture quality is set to medium or low ([#4621](https://github.com/cyberbotics/webots/pull/4621)).
```

File: docs/reference/changelog-r2022.md [hunk 2]
```diff
@@ -85,6 +62,24 @@ Released on XX XX, 2022.
...
+    - Removed the deprecated lua-gd library ([#4543](https://github.com/cyberbotics/webots/pull/4543)).
+    - Removed the 3D model import menu option with the introduction of the [CadShape](cadshape.md) node ([#4285](https://github.com/cyberbotics/webots/pull/4285)).
+    - Renamed `CameraRecognitionObject` and `ContactPoint` related API methods to use camel case ([#4477](https://github.com/cyberbotics/webots/pull/4477)).
+    - Renamed new controller and plugin specifications ([#4501](https://github.com/cyberbotics/webots/pull/4501)).
+  - Dependency Updates
+    - Upgraded to Qt6.2 on Windows, macOS and Linux ([#4189](https://github.com/cyberbotics/webots/pull/4189)).
+    - Removed Qt WebKit, WebChannel and WebEngine dependencies ([#4137](https://github.com/cyberbotics/webots/pull/4137)).
+    - Upgraded assimp from 5.0.1 to 5.2.3 on macOS and Linux ([#4285](https://github.com/cyberbotics/webots/pull/4285)).
+    - Linux: added support for Ubuntu 22.04 LTS and **stopped support for Ubuntu 18.04** ([#4502](https://github.com/cyberbotics/webots/pull/4502)).
+    - Linux: **removed support of Python 3.6 ([#4502](https://github.com/cyberbotics/webots/pull/4502))**.
```
Comment (ygoumaz) [inline docs/reference/changelog-r2022.md:81]:
```suggestion
    - Linux: **removed support of Python 3.6** ([#4502](https://github.com/cyberbotics/webots/pull/4502)).
```

**Review by ygoumaz — CHANGES_REQUESTED** [review]
Much cleaner this way, thank you! I have only spotted minor corrections to apply. I will add missing entries from my previous PRs.

**Comment by ThomasOliverKimble** [comment]
Applied corrections and merged develop, should be good now.

**Review by ygoumaz — APPROVED** [review]
Thank you!

**Pull request merged**